Microwave.Classes/Controllers/CookController.cs

```diff
@@ -47,17 +53,21 @@ public void StartCooking(int power, int time)
         public void Stop()
         {
             isCooking = false;
+            myBuzzer.CookingIsEndedSound();
```

Comment:
Skal der komme en lyd når man stopper for cooking? Er det ikke kun når timeren løber ud af der skal komme "Ding ding ding"